server/src/controllers/ChapterUser/resolver.ts

```diff
@@ -277,10 +300,18 @@ export class ChapterUserResolver {
       where: { user_id_chapter_id: { chapter_id: chapterId, user_id: userId } },
     });
 
+    const subject = `Your role in chapter has been changed to ${newChapterRole}`;
+    const body = `Hi ${chapterUser.user.name}.<br />
+    Your role in chapter has been changed to ${newChapterRole}.<br />
+    `;
```

Comment:
These are a touch repetitive. How about `Role changed in ${chapter.name}` as the title and `Your role in chapter ${chapter.name} has been changed from ${oldChapterRole} to ${newChapterRole}`? Something like that.